java/src/jmri/jmrix/can/cbus/CbusPowerManager.java

```diff
@@ -12,14 +12,16 @@
  * PowerManager implementation for controlling CBUS layout power.
  *
  * @author Bob Jacobsen Copyright (C) 2001
- * @author Andrew CRosland Copyright (C) 2009
+ * @author Andrew Crosland Copyright (C) 2009, 2021
  */
 public class CbusPowerManager extends AbstractPowerManager<CanSystemConnectionMemo> implements CanListener {
 
     private TrafficController tc;
+    private CanSystemConnectionMemo _memo;
 
     public CbusPowerManager(CanSystemConnectionMemo memo) {
         super(memo);
+        _memo = memo;
```

Comment:
Why do you need _memo in this class? Cannot you access memo from the super class? If the same value is in two different variables, the risk for one variable to be updated but not the other increases.